databases/core.py

```diff
@@ -113,7 +112,10 @@ async def disconnect(self) -> None:
             self._global_transaction = None
             self._global_connection = None
         else:
-            self._connection_context = ContextVar("connection_context")
+            current_task = asyncio.current_task()
+            assert current_task is not None, "No currently running task"
+            if current_task in self._connections:
+                del self._connections[current_task]
```

Comment:
This style of removal is fine with me! I have a minor preference for `self._connections.pop(current_task, None)` because it's shorter but I'd stick with whatever style you've seen in this repository.

Reply:
No one style seems more preferred here, in encode/starlette, encode/HTTPX, or encode/orm. The other three projects have about the same number of occurrences of `if x in map: del map[x]` as they do with `map.pop(x, None)`. Since neither wins out, I'm just going to leave this as is.